discovery_engine/test/repository/hive_active_document_repo_test.dart

```diff
@@ -84,18 +80,6 @@ Future<void> main() async {
         await repo.update(id1, data);
       });
 
-      test('existing smbert embedding', () async {
-        final emb = await repo.smbertEmbeddingById(id1);
-        expect(emb, equals(data.smbertEmbedding));
-      });
```

Comment:
I would keep the test to read the data from db, because we need that for migration. Other tests we can remove I think.